api/src/test/java/org/pmiops/workbench/access/UserServiceAccessTest.java

```diff
@@ -389,22 +389,26 @@ public void test_updateUserWithRetries_ducc_unbypassed_aar_expired_noncompliant(
         });
   }
 
-  // Publications confirmation is subject to annual renewal and cannot be bypassed.
+  // Publications confirmation is subject to annual renewal.
 
   @Test
-  public void test_updateUserWithRetries_publications_not_confirmed() {
+  public void test_updateUserWithRetries_publications_unbypassed_publications_not_confirmed() {
```

Comment:
Nit- suggestion: may be use: `doNotBypass` instead of `unbypassed`?

Reply:
"unbypassing" is the typical term we use to refer to the act of removing a bypass